### ISSUE TITLE
Can I use it without  the oculus sdk?

### ISSUE THREAD

**Issue by elblogbruno** [IanPhilips/UnityAndroidVRARBrowser#8]
Can I use it without  the oculus sdk? I'm planning to use it on android not oculus, and I'll really love to use it, maybe even showing the result if you want.
Nice work!

**Comment by IanPhilips** [comment]
unfortunately you need the sdk as it relies on the oculus plugin and OVRSurfaceOverlay

**Comment by IanPhilips** [comment]
Can you not include the oculus sdk in your android app anyways, though?

**Comment by elblogbruno** [comment]
I'm using my own raycaster and camera rig. Removing them makes the project doesn't work. And now i'm having an issue that says OVRPlugin not found.

**Comment by IanPhilips** [comment]
yes, you need OVRPlugin, but I don't see anything wrong with using your own raycaster and camera rig as long as you include the plugin

**Comment by elblogbruno** [comment]
well the thing is I haven't removed the ovrplugin. just the camera prefab.

**Comment by gtk2k** [comment]
+1
I want without the oculus sdk.

**Comment by DaveAccenture** [comment]
+1 also would like no Oculus sdk. Can the parts that truly require it be extracted from the Oculus SDK like the surface overlay?

**Comment by IanPhilips** [comment]
This library uses `OVROverlay` and this will only change if someone submits a PR with an alternative method. I doubt you can use this without the Oculus SDK, but if you're curious you should ask here: https://developer.oculus.com/support/

**Comment by Jamesfleming1** [comment]
Would it be able to work will the deprecated Oculus package? I am using the old one for a project but would like to include this in my project.

**Comment by wei-kris** [comment]
+1
I want without the oculus sdk.
I hope it can run on PicoVR

**Comment by RoxDevvv** [comment]
+1 
i also want without oculus sdk

**Comment by RoxDevvv** [comment]
well can you explain at least the purpose of OVROverlay so i could try to remove it by my self
any hints will be appreciated 
thank you

**Comment by IanPhilips** [comment]
It's been a while so I can't really accurately describe what it does but from what I remember It's the opengl surface that the gecko engine writes to. I tried to make my own opengl surface but I failed! I ended up having to use the overlay instead. You can read more about it here: https://developer.oculus.com/documentation/unity/unity-ovroverlay/

**Comment by xinczhang** [comment]
> +1 I want without the oculus sdk. I hope it can run on PicoVR

I successfully run it in pico vr. Very much thx to Ianphilips!
